internal/controllers/synthetic_devices_controller.go

```diff
@@ -235,25 +238,39 @@ func (vc *SyntheticDevicesController) MintSyntheticDevice(c *fiber.Ctx) error {
 		return fiber.NewError(fiber.StatusBadRequest, "Invalid signature provided")
 	}
 
-	err = vc.sendVirtualDeviceMintPayload(c.Context(), hash, req.VehicleNode, integration.TokenId, ownerSignature)
+	childKeyNumber := generateRandomNumber()
```

Comment:
I think we want to stop using this. I don't think there's a point to using `IDENTITY` if we're generating random stuff in Go. That does put us in a bit of a bind if we want the address to be non-null.

Reply:
I would use [`CREATE SEQUENCE`](https://www.postgresql.org/docs/current/sql-createsequence.html), how about that?